### PR TITLE
data:arpitghura

### PR DESCRIPTION
**Pull request by arpitghura**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2882"></a>

### CODE REVIEW

File: data/arpitghura.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      "icon": "FaYoutube",
+      "color": "#FF0000"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://linkedin.com/in/arpitghura",
+      "icon": "FaLinkedin"
+    }
+  ],
+  "events": [
```
Comment (amandamartin-dev) [inline data/arpitghura.json:49]: Events live in a slightly different location per the documentation here https://linkfree.eddiehub.io/docs/how-to-guides/events

Take a look and move the events portion into a separate file and you should be good here. Let us know if anything does not make sense

**Review by amandamartin-dev — CHANGES_REQUESTED** [review]

**Comment by arpitghura** [comment]
I have done the required changes @amandamartin-dev

File: data/arpitghura/events/2022-12-18-devfest.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "isInPerson": true,
+    "color": "green",
+    "name": "Google DevFest Bhopal",
+    "description": "This year, your DevFest will feature leading experts on Google technologies in areas like User Interfaces, Angular, Android Development, Building TV Apps by Flutter, SEO, Building a Chrome Extension, and so much including game activities like Guessing the Logo, and Guessing the Founder!",
+    "date": {
+        "start": "2022-12-18T10:00:00.000+00:00",
+        "end": "2022-12-18T17:30:00.000+00:00"
+    },
+    "url": "https://devfest.gdgbhopal.com"
+}
```
Comment (Panquesito7) [inline data/arpitghura/events/2022-12-18-devfest.json:12]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by amandamartin-dev — APPROVED** [review]

**Comment by amandamartin-dev** [comment]
checked my request, thanks for moving events to the new file.  Looks good!

**Review by Panquesito7 — APPROVED** [review]

**Review by eddiejaoude — APPROVED** [review]
Great collaboration 👍

**Pull request merged**